x-pack/platform/packages/shared/kbn-inference-endpoint-ui-common/src/components/configuration/adaptive_allocations_title.tsx

```diff
@@ -0,0 +1,97 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import React, { FC, useState } from 'react';
+import {
+  EuiButtonIcon,
+  EuiFlexGroup,
+  EuiFlexItem,
+  EuiPopover,
+  EuiPopoverTitle,
+  EuiSpacer,
+  EuiTitle,
+  EuiText,
+} from '@elastic/eui';
+import { FormattedMessage } from '@kbn/i18n-react';
+
+export const AdaptiveAllocationsTitle: FC = () => {
+  const [isPopoverOpen, setIsPopoverOpen] = useState(false);
+
+  return (
+    <>
+      <EuiFlexGroup alignItems="center" gutterSize="none">
+        <EuiFlexItem grow={false}>
+          <EuiTitle size="xxs" data-test-subj="max_number_of_allocations-details-label">
+            <h5>
+              <FormattedMessage
+                id="xpack.inferenceEndpointUICommon.components.adaptiveResourcesTitle"
+                defaultMessage="Adaptive resources"
+              />
+            </h5>
+          </EuiTitle>
+        </EuiFlexItem>
+        <EuiFlexItem grow={false}>
+          <EuiPopover
+            anchorPosition="upCenter"
+            button={
+              <EuiButtonIcon
+                color="text"
+                size="xs"
+                onClick={() => setIsPopoverOpen(!isPopoverOpen)}
+                iconType="iInCircle"
+                aria-label="Open documentation"
+              />
+            }
+            isOpen={isPopoverOpen}
+            closePopover={() => setIsPopoverOpen(false)}
+          >
+            <EuiPopoverTitle>
+              <FormattedMessage
+                id="xpack.inferenceEndpointUICommon.components.sectionPopoverTitle"
+                defaultMessage="Adaptive resources"
+              />
+            </EuiPopoverTitle>
+            <div style={{ width: '300px' }}>
+              <EuiText size="s">
+                <p>
+                  <FormattedMessage
+                    id="xpack.inferenceEndpointUICommon.components.sectionPopoverFirstParagraph"
+                    defaultMessage="The number of allocations scales automatically, based on load. Resources scale up when the load increases and scale down when the load decreases. We scale down to 0 <bold>{scaleDown}</bold> to the endpoint to <bold>{optimize}</bold>."
+                    values={{
+                      scaleDown: 'after 24 hours of no inference calls',
+                      optimize: 'optimize cost and performance',
+                      bold: (str) => <strong>{str}</strong>,
+                    }}
```

Comment:
I believe not all languages follow the same sentence structure as English, so I wonder if there might be difficulties translating the values.